### ISSUE TITLE
deprecate (or automate) GitHub releases

### ISSUE THREAD

**Issue by idiomatic** [idiomatic/HandyNotes_Achievements#33]
- [ ] decide if https://github.com/idiomatic/HandyNotes_Achievements/releases is relevant or redundant

either:
- [ ] add GitHub actions to tag and release to GitHub in tandem with CurseForge
- [ ] or ignore or remove old archives

**Comment by JanGalek** [comment]
We can automate it with https://github.com/marketplace/actions/create-a-release , https://github.com/thomaseizinger/keep-a-changelog-new-release and https://github.com/ffurrer2/extract-release-notes

source: https://riggaroo.dev/using-github-actions-to-automate-our-release-process/

**Comment by JanGalek** [comment]
It will need small refactore change log (change to standard - https://keepachangelog.com/en/1.0.0/)

**Comment by idiomatic** [comment]
Interesting.  However, I doubt there's actually a demand for GitHub-hosted release archives.

* GitHub probably doesn't have a downloader like the Curse Client
* GitHub does not show number-of-downloads like CurseForge
* GitHub does not enforce alpha/beta/release categorization as firmly as CurseForge

I advise de-prioritizing this issue.  Tasks that add unnecessary complexity or dependence should not be rushed.

**Comment by JanGalek** [comment]
- github has client - https://desktop.github.com/
- there is util (badges) for show it (https://shields.io/category/downloads)

![GitHub Releases](https://img.shields.io/github/downloads/idiomatic/HandyNotes_Achievements/v0.5.0/total?style=social)

- thats true there is just release and pre-release

but I understand ;)

**Comment by idiomatic** [comment]
> * github has client - https://desktop.github.com/

GitHub Desktop [client] is more a Git front end than an add-on release auto-updater.  I would be thrilled if we could get WoW players to switch from [specialty add-on manager clients](https://www.wowhead.com/guides/best-addon-managers-wow-shadowlands) to developer-oriented, submodule-savvy clients (like `git`, [GitHub Desktop](https://desktop.github.com/), [Tower](https://www.git-tower.com/mac), or my current favorite [Fork](https://git-fork.com/)), but that's counter-culture.

> * there is util (badges) for show it (https://shields.io/category/downloads)
> 
> ![GitHub Releases](https://camo.githubusercontent.com/9079c57e8e1400e0cf78da1685d23b43b1fb4385/68747470733a2f2f696d672e736869656c64732e696f2f6769746875622f646f776e6c6f6164732f6964696f6d617469632f48616e64794e6f7465735f416368696576656d656e74732f76302e352e302f746f74616c3f7374796c653d736f6369616c)

TIL.

**Comment by JanGalek** [comment]
Yeah. It sounds good :). I have plan create new wow addon manager (reason ? I'm linux user and twitch app work on wine good but it's not perfect, so multi-os).

**Comment by idiomatic** [comment]
Note that the GitHub-hosted releases were appropriate *before* GitHub Actions existed and had hosted artifacts.  Also, locally-assembled releases was more tolerable when this project had a single developer with a predictable toolchain.

Luckily we've evolved, and should move forwards.

**Comment by idiomatic** [comment]
> new wow addon manager

If that's what @wowads is, I'd recommend a different name.  "Ads" is not short for "add-ons", but "advertisements".

**Comment by JanGalek** [comment]
Thanks for advice :), I renamed it to @wowmua :) https://github.com/idiomatic/HandyNotes_Achievements/pull/36
